test/filtercontainer.cpp

```diff
@@ -0,0 +1,122 @@
+#include "filtercontainer.h"
+
+#include "3rd-party/catch.hpp"
+#include "confighandlerexception.h"
+
+using namespace newsboat;
+
+TEST_CASE("FilterContainer::handle_action handles `define-filter`",
+	"[FilterContainer]")
+{
+	FilterContainer filters;
+
+	const std::string action = "define-filter";
```

Comment:
Nitpick: Similar places below use `auto` instead of `std::string`

Reply:
Fixed. Since it's a trivial change, I force-pushed the new commits.